custom_components/hwam_stove/translations/nl.json

```
@@ -0,0 +1,202 @@
+        "name": "Klep niet verbonden"
+      },
+      "safety_alarms_valve_calibration_error": {
+        "name": "Klep kalibratie"
+      },
+      "safety_alarms_stove_overheat": {
+        "name": "Oververhitting"
+      },
+      "safety_alarms_door_open_too_long": {
+        "name": "Deur te lang open",
+        "state": {
+          "off": "Nee",
+          "on": "Ja"
+        }
+      },
+      "safety_alarms_manual_safety_alarm": {
+        "name": "Handmatige veiligheid"
+      },
+      "safety_alarms_stove_sensor_fault": {
+        "name": "Sensoren"
+      }
+    },
+    "button": {
+      "start": {
+        "name": "Start"
+      },
+      "sync_clock": {
+        "name": "Synchroniseer klok"
+      }
+    },
+    "datetime": {
+      "date_and_time": {
+        "name": "Datum en tijd"
+      }
+    },
+    "number": {
+      "burn_level": {
+        "name": "Brandniveau"
+      }
+    },
+    "sensor": {
+      "algorithm": {
+        "name": "Algoritme"
+      },
+      "message_id": {
+        "name": "Bericht ID"
+      },
+      "new_firewood_estimate": {
+        "name": "Schatting nieuw brandhout"
+      },
+      "night_lowering": {
+        "name": "Nacht verlaging",
+        "state": {
+          "disabled": "Uitgeschakeld",
+          "init": "Initialiseren",
+          "on_day": "Dag",
+          "on_night": "Nacht",
+          "on_manual_night": "Handmatig nacht"
+        }
+      },
+      "operation_mode": {
+        "name": "Operationele status",
+        "state": {
+          "init": "Initialiseren",
+          "self_test": "Zelftest",
+          "normal": "Normaal",
+          "temperature_fault": "Temperatuurefout",
+          "o2_fault": "Zuurstoffout",
+          "calibration": "Kalibratie",
+          "safety": "Veiligheid",
+          "manual": "Handmatig",
+          "motor_test": "Motor est",
+          "slow_combustion": "Langzame verbranding",
+          "low_voltage": "Lage spanning"
+        }
+      },
+      "oxygen_level": {
+        "name": "Zuurstof niveau"
+      },
+      "phase": {
+        "name": "Fase",
+        "state": {
+          "ignition": "Ontsteking",
+          "burn": "Verbranding",
+          "glow": "Gloeien",
+          "standby": "Stand-by"
+        }
+      },
+      "room_temperature": {
+        "name": "Kamer temperatuur"
+      },
+      "stove_temperature": {
+        "name": "Kachel temperatuur"
+      },
+      "time_since_remote_message": {
+        "name": "Tijd sinds laatste bericht"
+      },
+      "time_to_new_firewood": {
+        "name": "Tijd tot nieuw brandhout"
+      },
+      "valve_1_position": {
+        "name": "Klep 1 positie"
+      },
+      "valve_2_position": {
+        "name": "Klep 2 positie"
+      },
+      "valve_3_position": {
+        "name": "Klep 3 positie"
+      }
+    },
+    "switch": {
+      "night_lowering": {
+        "name": "Nacht verlaging"
+      },
+      "remote_refill_alarm": {
+        "name": "Bijvul alarm"
```

```suggestion
        "name": "Bijvulalarm"
```
One word? Although both versions look strange to me :sweat_smile:

---

Translating in just a word is way harder then i thought :-) But couldn't come up with a better word. The translation sites come up with: navulalarm, bijvulalarm of hervulalarm. When it is on, my google home says "kachel bijvullen". So I chose the bijvul alarm.